### ISSUE TITLE
Export fails for big exports in zip mode

### ISSUE THREAD

**Issue by sphuber** [aiidateam/aiida-core#2358]
```
Traceback (most recent call last):
  File "/home/aiida/.virtualenvs/aiida_sdb/bin/verdi", line 9, in <module>
    sys.exit(verdi())
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/aiida/.virtualenvs/aiida_sdb/local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/aiida/code/aiida/env/sdb/aiida-core/aiida/cmdline/commands/cmd_export.py", line 132, in create
    export_function(entities, outfile=output_file, **kwargs)
  File "/home/aiida/code/aiida/env/sdb/aiida-core/aiida/orm/importexport.py", line 2621, in export_zip
    export_tree(what, folder=folder, silent=silent, **kwargs)
  File "/home/aiida/code/aiida/env/sdb/aiida-core/aiida/orm/importexport.py", line 2367, in export_tree
    fhandle.write(json.dumps(data))
  File "/home/aiida/code/aiida/env/sdb/aiida-core/aiida/orm/importexport.py", line 2497, in __exit__
    self.close()
  File "/home/aiida/code/aiida/env/sdb/aiida-core/aiida/orm/importexport.py", line 2489, in close
    self._zipfile.writestr(self._fname, self._buffer.read())
  File "/usr/lib/python2.7/zipfile.py", line 1236, in writestr
    zinfo.CRC = crc32(bytes) & 0xffffffff       # CRC-32 checksum
OverflowError: size does not fit in an int
```

**Comment by CasperWA** [comment]
This seems to be related to the (solved) Python issues [here](https://bugs.python.org/issue27130) and [here](https://bugs.python.org/issue23306). But do we need to use `zlib` perhaps - or make sure a minimum version of Python 2.7 is used?

**Comment by ltalirz** [comment]
@sphuber The underlying zlib issue seems to have been addressed in python 3.5+ - can you check whether this works in AiiDA now?

**Comment by ltalirz** [comment]
Given that this as not resurfaced, I assume this was resolved by switching to a recent python version.
Feel free to reopen if the issue resurfaces.
